### ISSUE TITLE
Mandatory field 'editor' for 'inproceedings'?

### ISSUE THREAD

**Issue by AlexanderWillner** [plk/biber#104]
Version: 2.2

Steps that will reproduce the problem?
1. ```biber -T -V file```

What is the expected result?
No warnings.

What happens instead?
```WARN - Datamodel: Entry 'Strassner2002' (thesis.bib): Missing mandatory field 'editor'```

Any additional information:
Based on http://www.lsv.ens-cachan.fr/~markey/BibTeX/doc/ttb_en.pdf (page 13) the field ```editor``` is not required for ```in proceedings```.

Content of ```file.bib```
```
@inproceedings{Strassner2002,
author = {Strassner, J.},
booktitle = {Network Operations and Management Symposium (NOMS)},
doi = {10.1109/NOMS.2002.1015622},
isbn = {0-7803-7382-0},
language = {English},
pages = {753--766},
publisher = {IEEE},
title = {{DEN-ng: achieving business-driven network management}},
url = {http://ieeexplore.ieee.org/xpls/abs{\_}all.jsp?arnumber=1015622},
year = {2002}
}
```

**Comment by plk** [comment]
Hmm, yes, seems to be an inconsistency in the default data model. This actually needs to be fixed in biblatex's default datamodel which it passes to biber. This is done in the 3.3 DEV branch of biblatex.

**Comment by dschrempf** [comment]
Sorry to bump this thread, but I am still having these warnings, and in the biblatex documentation the editor field is optional, so the inconsistency is still there.

I have some bibliography entries with inproceedings, where there is no editor. What can I do to remove the warnings?

**Comment by schoeps** [comment]
This is still an issue in biber version: 2.15 (beta). The biblatex manual says
> A single-volume conference proceedings. This type is very similar to @collection. It supports an optional organization field which holds the sponsoring institution. The editor is omissible in terms of § 2.3.2.
> Required fields: title, year/date

However biber-tool.conf requires the `editor` field
```
    <constraints>
      <entrytype>proceedings</entrytype>
      <entrytype>mvproceedings</entrytype>
      <constraint type="mandatory">
        <field>editor</field>
        <field>title</field>
      </constraint>
    </constraints>
    <constraints>
      <entrytype>inproceedings</entrytype>
      <constraint type="mandatory">
        <field>author</field>
        <field>editor</field>
        <field>title</field>
        <field>booktitle</field>
      </constraint>
    </constraints>
```

**Comment by plk** [comment]
This has been corrected in 2.15 DEV and also in biblatex 3.15 DEV branch which also had incorrect defaults.
